core/src/main/java/org/mule/DefaultMuleMessage.java

```diff
@@ -1178,9 +1181,19 @@ public void setEncoding(String encoding)
     }
 
     /**
-     * @param mimeType
-     * @since 3.0
+     * {@inheritDoc}
+     */
+    @Override
+    public String getMimeType()
```

Comment:
I think we talked about this method... but... why do you need to add this getter instead of using the datatype?